### PR TITLE
IMPLEMENT A PLUGIN-INFO ENDPOINT. 

### PR DESCRIPTION
**Pull request by Adeniyii**
- The info JSON response has the correct fields, but is filled with dummy data for now.
- Changes will be made accordingly.

### CODE REVIEW

File: routes/infoRoute.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+const { Router } = require('express');
+
+const router = Router();
+
+/**
+ * GET info about the goals plugin.
+ * @param {request} req Express request object.
+ * @param {response} res Express response object.
+ */
+const getPluginInfo = (req, res) => {
```
Comment (Asylcreek) [inline routes/infoRoute.js:12]: Please extract this function into a controller

**Review by Asylcreek — CHANGES_REQUESTED** [review]

File: routes/infoRoute.js (hunk 1 above)
Comment (Adeniyii) [reply routes/infoRoute.js:12]: I'm on it.

**Comment by Asylcreek** [comment]
Alright then. Please pull from the staging branch and push again. It seems like you are a commit behind

**Comment by Adeniyii** [comment]
close #329

**Pull request merged**